leftwm/src/bin/leftwm-check.rs

```diff
@@ -76,29 +95,65 @@ pub fn load_from_file(fspath: Option<&str>, verbose: bool) -> Result<Config> {
             println!("\x1b[1;35mNote: Using file {} \x1b[0m", fspath);
             PathBuf::from(fspath)
         }
-
-        None => BaseDirectories::with_prefix("leftwm")?.place_config_file("config.toml")?,
+        None => {
+            let ron_file =
+                BaseDirectories::with_prefix("leftwm")?.place_config_file("config.ron")?;
+            let toml_file =
+                BaseDirectories::with_prefix("leftwm")?.place_config_file("config.toml")?;
+            if Path::new(&ron_file).exists() {
+                ron_file
+            } else if Path::new(&toml_file).exists() {
+                println!(
+                    "\x1b[1;93mWARN: TOML as config format is about to be deprecated.
+      Please consider migrating to RON or contact the theme creator about this topic.\x1b[0m"
```

Comment:
Should probably change this to "Please consider migrating config.toml using leftwm-check -m" owtte